ocs_ci/deployment/aws.py

```diff
@@ -88,9 +88,13 @@ class AWSIPI(AWSBase):
     def __init__(self):
         self.name = self.__class__.__name__
         super(AWSIPI, self).__init__()
+        force_download = (
+            config.RUN['cli_params'].get('deploy')
+            and config.DEPLOYMENT['force_download_client']
```

Comment:
@petr-balogh and config.DEPLOYMENT['force_download_installer']?

Reply:
Yep, forgot to change when copy paste. Thanks. Fixed